### ISSUE TITLE
agent attribute GitHub should be public

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#8656]
Help us understand your request (check below):

- [x] other

**Describe what you're trying to do** 

https://arctos.database.museum/info/ctDocumentation.cfm?table=ctagent_attribute_type#github is private. It should not be. I think this must be a mistake, and will change it unless someone with some compelling reason to maintain this status surfaces in the next few days.

**Comment by mkoo** [comment]
I dont see a reason not to have it fully public (it is already for some), especially since our public forum is, well, public! Thanks!
